### DISCUSSION TITLE
Using on a wall mounted iPad

### DISCUSSION THREAD

**Discussion by dannyuk1982** [dgreif/ring#1494] [Q&A]
Hi, I can't find much anywhere about how to solve this so thought I'd ask here.

I have a wall mounted iPad which runs mainly HomeKit in kiosk mode (aka Guided Access), It's right next to a Ring keypad which can be used to disarm the alarm.

**The problem**
This setup works great other than anyone can just disarm the alarm with a couple of taps. 

**Potential solutions:**
- Hide the alarm from Homekit, this is no good as I arm/disarm the alarm using location based automations so Homekit needs to "see" the alarm
- Just lock the iPad and use the iOS passcode, this seems to be the best way so far, but it's a pain - means I can't use kiosk/Guided Access mode and undoes the convenience of location based automations and otherwise "just working".

Has anyone got any solutions or suggestions, or have you got a similar use case?

**Comment by smockle** [comment]
iPadOS 18 (released yesterday) includes the ability to [“Customize Access”](https://support.apple.com/guide/iphone/invite-others-to-control-accessories-iphcbaf7e8f3/ios#iph5c179e9cc). If your wall-mounted iPad is using a separate Apple ID, you could invite that Apple ID as a guest to your HomeKit home and limit its access to security systems.

**Reply by dannyuk1982** [reply]
Ah thank you, that looks like a great solution! My iPad can't upgrade to iOS 18 but hopefully it's enough to have it on an iPhone to set. Cheers!

**Reply by dannyuk1982** [reply]
Updated my Apple TV and iPhone to tvOS/iOS 18, but alas the guest device has to be on 18 too 🤦‍♂️ although it happily shares resident homekit access with the iPadOS 17.7 iPad, but not guest
